nebula/ui/components/VPNControllerView.qml

```diff
@@ -649,6 +662,7 @@ Item {
                 height: iconSize
                 width: iconSize
             }
+            opacity: parent.enabled ? 1 : .6
```

Comment:
definitely not blocking, but I think we should get a tech debt bug written to standardize the opacity of a disabled icon button throughout the codebase (and maybe even standardize the way we add images to `VPNIconButton` - some instances use `Image`, others use `VPNIcon`, etc which opens the door for inconsistency).

For instance, [in VPNMenu](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/70df7ff2d46a652e4b8229f4e78bca8b3740cc84/nebula/ui/components/VPNMenu.qml#L59) we are using `.4` opacity for a disabled button

Reply:
+1